relay/engines/docker.go

```diff
@@ -1,38 +1,40 @@
 package engines
 
 import (
-	"bytes"
-	"encoding/json"
 	"errors"
 	"fmt"
 	log "github.com/Sirupsen/logrus"
-	"github.com/fsouza/go-dockerclient"
+	"github.com/docker/engine-api/client"
+	"github.com/docker/engine-api/types"
+	"github.com/docker/engine-api/types/container"
+	"github.com/docker/engine-api/types/filters"
+	"github.com/operable/circuit"
 	"github.com/operable/go-relay/relay/config"
-	"github.com/operable/go-relay/relay/messages"
+	"golang.org/x/net/context"
+	"io/ioutil"
+	"net/http"
 	"strings"
-	"time"
 )
 
-var errorDockerDisabled = errors.New("Docker engine is disabled")
+const (
+	megabyte = 1024 * 1024
+)
 
-var relayCreatedLabel = "io.operable.cog.relay.created"
-var relayCreatedFilter = "io.operable.cog.relay.created=yes"
+var relayCreatedLabel = "io.operable.cog.relay.create"
+var errorDriverImageUnavailable = errors.New("Command driver image is unavailable")
 
 // DockerEngine is responsible for managing execution of
 // Docker bundled commands.
 type DockerEngine struct {
-	client      *docker.Client
-	relayConfig config.Config
-	config      config.DockerInfo
-	stdout      bytes.Buffer
-	stderr      bytes.Buffer
+	client        *client.Client
+	relayConfig   *config.Config
+	config        config.DockerInfo
+	registryToken string
+	cache         *envCache
 }
 
 // NewDockerEngine makes a new DockerEngine instance
-func NewDockerEngine(relayConfig config.Config) (Engine, error) {
```

Comment:
In the case described above, relay will emit log entries to indicate what's happening:

```
INFO[2016-06-16T10:25:24-04:00] Skipping Docker-based bundle format 0.2.15.
INFO[2016-06-16T10:25:24-04:00] Skipping Docker-based bundle statuspage 0.1.5.
```